modules/platforms/cpp/core-test/src/cache_test.cpp

```diff
@@ -132,6 +132,43 @@ struct CacheTestSuiteFixture
     }
 };
 
+/*
+ * Test setup fixture.
+ */
+struct CacheNativePersistenceTestSuiteFixture
+{
+    /* Nodes started during the test. */
+    Ignite grid0;
+
+    /** Cache accessor. */
+    cache::Cache<int, int> Cache()
+    {
+        return grid0.GetCache<int, int>("partitioned");
+    }
+
+    /*
+     * Constructor.
+     */
+    CacheNativePersistenceTestSuiteFixture()
+    {
```

Comment:
I think we should use `ClearLfs()` utility method here. You can copy it from thin client tests or ODBC tests.

Reply:
Added.